src/main/resources/com/google/api/codegen/py/setup.py.snip

```diff
@@ -38,7 +39,7 @@
       ],
       description='GAPIC library for the {@metadata.fullName}',
       include_package_data=True,
-      long_description=open('README.rst').read(),
+      long_description=io.open('README.rst', encoding='utf-8').read(),
```

Comment:
This does not close the file handler. This matters because calls to `pkg_resources.get_distribution` will read this file.

Above the `setup(...)` call, we need:

```python
with io.open('README.rst', 'r', encoding='utf-8'): as readme_file:
    long_description = readme_file.read()
```

And then just use the `long_description` variable here.

Reply:
Done.